src/ol/style/iconstyle.js

```diff
@@ -456,8 +456,8 @@ ol.style.IconImage_.get = function(image, src, size, crossOrigin, imageState) {
  */
 ol.style.IconImage_.prototype.determineTainting_ = function() {
   var context = ol.dom.createCanvasContext2D(1, 1);
-  context.drawImage(this.image_, 0, 0);
   try {
+    context.drawImage(this.image_, 0, 0);
```

Comment:
I did try to keep it outside first, but I still got problems in IE11 then. At the moment I had an idea of the reason for this, but it slipped my mind now... But to answer your question, yes I had to move the drawImage into the try/catch block.